### DISCUSSION TITLE
Discord invite link?

### DISCUSSION THREAD

**Discussion by YagaoDirac** [MaxRobinsonTheGreat/EvolutionSimulatorV2#8] [General]
Any discord server?

**Comment by MaxRobinsonTheGreat** [comment]
No discord for the moment, I'd prefer to keep the discussion here. May change eventually.

**Comment by CyberPoincare** [comment]
these days we talk about LifeEngine in this channel: https://discord.com/channels/723535295680610344/735561190201360508
